packages/backend-api/src/api/util/queryComments.ts

```diff
@@ -36,21 +36,21 @@ export async function filterTopScoresByTaggingSensitivity(maxScores: ITopScores,
   const fetchComment = getComment || (async (commentId: string): Promise<ICommentInstance> => {
     const id = parseInt(commentId, 10);
 
-    return await Comment.findById(
+    return (await Comment.findById(
       parseInt(maxScores[id].commentId.toString(), 10),
       { include: [Article] },
-    );
+    ))!;
```

Comment:
I think I prefer full explicit style of type coercion (`as any as <intended type>`) as it makes these things easier to spot and think about. Hopefully in time we can type-parameterize `findById`, or similar to move these also and make code more run-time checked.

Reply:
The '!' is removing the " | null" from the type.  Later versions of Sequelize types change the type signature of findById to be:
    public static findById<M extends Model>(
        this: { new (): M } & typeof Model,
        identifier?: number | string,
        options?: FindOptions
    ): Promise<M | null>
    public static findById<M extends Model>(
        this: { new (): M } & typeof Model,
        identifier: number | string,
        options: NonNullFindOptions
    ): Promise<M>
Looking at that, we can, alternatively, fix by adding "rejectOnEmpty=true".  (It's quite funky that typescript allows for that:-) But I'm not sure it buys you anything - they will both result in an exception if the id is not valid, and AFAICT in all the cases here, the commentId is guaranteed to be valid, unless something has gone really wrong.